sentry-samples/sentry-samples-android/build.gradle.kts

```diff
@@ -109,4 +123,13 @@ dependencies {
     implementation(Config.Libs.retrofit2Gson)
 
     debugImplementation(Config.Libs.leakCanary)
+
+    androidTestImplementation("androidx.test:core-ktx:1.3.0")
+    androidTestImplementation("androidx.test:runner:1.3.0")
+    androidTestImplementation(Config.TestLibs.androidxJunit)
+    androidTestImplementation("androidx.test.espresso:espresso-core:3.3.0")
+    androidTestImplementation("com.android.support.test.uiautomator:uiautomator-v18:2.1.3")
+    //androidTestUtil("androidx.test:orchestrator:1.3.0")
+    androidTestImplementation("com.microsoft.appcenter:espresso-test-extension:1.4")
```

Comment:
I beleive a few of them already exist under the `Config` class, we can move them there or just use it if its existent